packages/cojson/src/sync.ts

```diff
@@ -403,27 +403,39 @@ export class SyncManager {
     }
 
     if (entry.state.type === "loading") {
-      const value = await entry.getCoValue();
```

Comment:
Probably just removing `await` keyword would help :)

Reply:
?? it wouldn't, everything in the then() still needs to happen after the covalue is loaded. (The diff view is very confusing unfortunately)